src/deprecated.jl

```diff
@@ -1,5 +1,34 @@
 import Base: @deprecate
 
+@deprecate DataFrame(t::Type, nrows::Integer, ncols::Integer) DataFrame(Matrix{t}(undef, nrows, ncols))
+
+@deprecate DataFrame(column_eltypes::AbstractVector{<:Type},
+                     nrows::Integer) DataFrame(column_eltypes, gennames(length(column_eltypes)), nrows)
```

Comment:
`gennames` isn't available outside of `DataFrames`. I guess `'x' .* string.(1:nrows)` is OK?

Reply:
fixed (the code needs to be different, but the approach is right 😄)